docs/en_US/TrainingService/AMLMode.md

```diff
@@ -81,6 +81,13 @@ amlConfig:
     * optional key, default 1. Used to specify the max concurrency trial number on a GPU device.
 * useActiveGpu
     * optional key, default false. Used to specify whether to use a GPU if there is another process. By default, NNI will use the GPU only if there is no other active process in the GPU.
+* machineScheduler
```

Comment:
It shouldn't be a AML only feature, may be useful for other platforms.

Reply:
@QuanluZhang @squirrelsc  we need to discuss the scenario of this feature in the meeting, to set this single schuduler only for aml or for all of platforms. To expose the configuration or set it as internally.